src/coreclr-debug/activate.ts

```diff
@@ -124,14 +124,20 @@ export async function registerAdapterExecutionCommand(channel: vscode.OutputChan
     let util = new CoreClrDebugUtil(common.getExtensionPath(), logger);
     let success = true;
 
-    if (!CoreClrDebugUtil.existsSync(util.installCompleteFilePath())) {
```

Comment:
I think you want to first check if the debugger file exists. This way in the warm case, you don't need to check if two files exist.